data/json/npcs/BGSS_talk_common.json

```diff
@@ -0,0 +1,965 @@
+      { "text": "<done_conversation_section>", "topic": "TALK_FRIEND" },
+      { "text": "<end_talking>", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "BGSS_OUT_OF_TOWN_2_STORY2",
+    "type": "talk_topic",
+    "dynamic_line": "By morning, the area around the lake was covered in a pinkish mold, and there were walking mushrooms around shooting clouds of the dust in the air.  We didn't know what was going on, but neither of us wanted to stay and find out.  We packed up our shit, scraped off the boat, and took off upriver.",
+    "responses": [
+      { "text": "What happened to your friend?", "topic": "BGSS_OUT_OF_TOWN_2_STORY3" },
+      { "text": "<done_conversation_section>", "topic": "TALK_FRIEND" },
+      { "text": "<end_talking>", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "BGSS_OUT_OF_TOWN_2_STORY3",
+    "type": "talk_topic",
+    "dynamic_line": "She took sick a few hours after we left the lake.  Puking, complaining about her joints hurting.  I took us to a little shop I knew about on the riverside, hoping they might have something to help or at least know what was going on.",
+    "responses": [
+      { "text": "I guess they didn't know.", "topic": "BGSS_OUT_OF_TOWN_2_STORY4" },
+      { "text": "<done_conversation_section>", "topic": "TALK_FRIEND" },
+      { "text": "<end_talking>", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "BGSS_OUT_OF_TOWN_2_STORY4",
+    "type": "talk_topic",
+    "dynamic_line": "The shop was empty, actually.  She was desperate though, so I broke in.  I found out more about the chaos in towns from the store radio.  Got my friend some painkillers and gravol, but when I came out to the boat, well...  it was too late for her.",
+    "responses": [
+      { "text": "She was dead?", "topic": "BGSS_OUT_OF_TOWN_2_STORY5" },
+      { "text": "<done_conversation_section>", "topic": "TALK_FRIEND" },
+      { "text": "<end_talking>", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "BGSS_OUT_OF_TOWN_2_STORY5",
+    "type": "talk_topic",
+    "dynamic_line": "I wish.  That would have been a mercy.  She was letting out an awful, choking scream, and her body was shredding itself apart.  Mushrooms were busting out of every part of her.  I...  I ran.  Now I wish that I'd put her out of her misery, but going back there now would be suicide.",
+    "//": "In the future this NPC might give you the coordinates of his friend where you can go take down a fungal grove of some kind.",
+    "responses": [
+      { "text": "That's awful.  <done_conversation_section>", "topic": "TALK_FRIEND" },
+      { "text": "That's awful.  <end_talking>", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "BGSS_LOST_PARTNER_1_STORY1",
+    "type": "talk_topic",
+    "dynamic_line": [
+      {
+        "npc_female": "My husband made it out with me, but got eaten by one of those <swear> plant monsters a few days before I met you.  This hasn't been a great year for me.",
```

Comment:
Maybe we should add `<spouse>` tag for wife/husband? Or you can use `spouse` here (well maybe not so good suggestion for languages other than English).

Reply:
Do you mean instead of a version for male or female speakers?

I originally had it written gender neutral, but it sounded very strange. In the future I'm hoping we can make snippets conditional based on NPC characteristics, in which case I'll add snippets like <spouse> that automatically choose, but I don't think we're there yet